### DISCUSSION TITLE
AL should auto on if night mode is on

### DISCUSSION THREAD

**Discussion by filikun** [basnijholt/adaptive-lighting#338] [Ideas]
I have this struggle where the family turns off AL and manually controls the light and then at night time our night mode is automatically turned on but as some of the lights are have their AL turned off them lights won't apply night mode. I would love to be able to simply have an option to make the night light input Boolean act as a override and turn on AL as well.

**Comment by basnijholt** [comment]
Just make an automation that triggers when sleep mode is turned on. 😄
